src/interfaces/IMorpho.sol

```diff
@@ -130,6 +130,8 @@ interface IMorphoBase {
     /// @dev Either `assets` or `shares` should be zero. Most usecases should rely on `assets` as an input so the caller
     /// is guaranteed to have `assets` tokens pulled from their balance, but the possibility to mint a specific amount
     /// of shares is given for full compatibility and precision.
+    /// @dev When withdrawing all supply shares, the supply share price instantly resets to
+    /// `VIRTUAL_ASSETS`:`VIRTUAL_SHARES`.
```

Comment:
```suggestion
    /// @dev If the supply of a market gets depleted, the supply share price instantly resets to
    /// `VIRTUAL_ASSETS`:`VIRTUAL_SHARES`.
```
because:
- it's weird to only have this comment on supply and talk about withdrawing
- it can happen that there are very few shares and no assets, and in effect the ratio is reset to the initial ratio